### PR TITLE
TW-613: NFT details, Personal, button Burn NFT

### PR DESCRIPTION
**Pull request by svjatoslavtt**

### CODE REVIEW

File: src/modals/collectible-modal/collectible-modal.tsx [hunk 1]
```diff
@@ -151,13 +157,24 @@ export const CollectibleModal = () => {
...
           )}
+
+          <View style={styles.burnContainer}>
+            <TouchableWithAnalytics
+              Component={TouchableOpacity}
+              onPress={burnCollectible}
+              style={styles.burnButton}
+              testID={CollectibleModalSelectors.burnButton}
+            >
+              <Text style={styles.burnButtonText}>Burn Nft</Text>
```
Comment (Boltman92) [inline src/modals/collectible-modal/collectible-modal.tsx:174]: as I see, on the design this text is uppercase

**Review by herkoss — APPROVED** [review]

File: src/modals/collectible-modal/collectible-modal.tsx (hunk 1 above)
Comment (svjatoslavtt) [reply src/modals/collectible-modal/collectible-modal.tsx:174]: typography that I use has 'uppercase' style

Comment (Boltman92) [reply src/modals/collectible-modal/collectible-modal.tsx:174]: sorry, miss it

**Review by Boltman92 — APPROVED** [review]

**Review by vladvn1905 — APPROVED** [review]
Done build 1.14_669 ios

**Pull request merged**